WebRoot/static/dhtml/main.css

```diff
@@ -738,19 +1013,6 @@ input[type=search]::-webkit-search-results-decoration {
     -moz-appearance: none;
```

Comment:
This line is not going to be reached by Firefox because your class only target Chrome and Safari (webkit)